contracts/src/v0.8/automation/dev/v2_3/AutomationRegistry2_3.sol

```diff
@@ -325,6 +329,7 @@ contract AutomationRegistry2_3 is AutomationRegistryBase2_3, OCR2Abstract, Chain
     {
       Transmitter memory transmitter;
       address temp;
```

Comment:
couldn't we just reuse the above 'transmitterAddress' var?

Reply:
done